src/CMakeLists.txt

```diff
@@ -828,6 +846,8 @@ else()
     endif()
 endif()
 
+
+
```

Comment:
1 blank line is enough to split code blocks here and in other places too

Reply:
Fixed spacing issues in src/CMakeLists.txt